### DISCUSSION TITLE
Mine with multiple accounts.

### DISCUSSION THREAD

**Discussion by Glass47** [RakambdaOrg/ChannelPointsMiner#252] [Ideas]
Hello, can you please tell me how do i make this work with multiple accounts instead having to run 10 different instances, it's just ridiculous, if there is a way to put accounts in one file from which the miner reads and uses them to mine. 
Or would in config.json work if i add numbers so it know that i separated them?

```
 "accounts": [
      {
        "username1": "Acount1username",
        "password1": "passForAccount1",        
        "username2": "Acount2username",
        "password2": "passForAccount2",
        "username3": "Acount3username",
        "password3": "passForAccount3",
```

Example 
![image](https://user-images.githubusercontent.com/55170107/185641313-40aed043-1852-4501-9cbf-358a1a4e0c61.png)

Update: Seems like that this way doesn't work. Can you please implement some more piratical way of adding accounts? for example format i gave or just some better way. And lets say you if you are first time logging in it would require verification code(2fa) it should put all other accounts in queue so after you write 2Fa for account1, it goes to next and etc.. Example. 

Solution for this is just to copy paste and create multiple copies of same files.. Then you just put different accounts in each... It's really annoying that way, definitely solution but really bad and takes tons of time..

**Comment by Rakambda** [comment]
This is already supported:

```json
{
"accounts": [{
    ...
  },{
    ...
  },{
    ...
  }
]}
```

Replace each three dots with one account settings.

Would do something like:
```json
{
  "accounts": [
    {
      "username": "username",
      "password": "password",
      "streamerConfigDirectories": [
        {
          "path": "streamers",
          "recursive": true
        }
      ]
    },
    {
      "username": "username2",
      "password": "password2",
      "streamerConfigDirectories": [
        {
          "path": "streamers2",
          "recursive": true
        }
      ]
    },
    {
      "username": "username3",
      "password": "password3",
      "streamerConfigDirectories": [
        {
          "path": "streamers",
          "recursive": true
        }
      ]
    }
  ]
}
```

**Reply by Glass47** [reply]
Thank you for this, can you accept my friend request on discord? it would be easier for me to get better understanding how to make this work, its just showing reds yellows, i dont have much knowledge about programming because i can't ask anyone to explain me

**Reply by Rakambda** [reply]
I probably discarded it as it was out of nowhere, send it again.
